repose-aggregator/tests/performance-tests/roles/gatling/files/gatling.conf

```diff
@@ -0,0 +1,21 @@
+#########################
+# Gatling Configuration #
+#########################
+
+gatling {
+  charting {
+    indicators {
+      percentile1 = 50      # Value for the 1st percentile to track in the reports, the console summary and Graphite
+      percentile2 = 95      # Value for the 2nd percentile to track in the reports, the console summary and Graphite
+      percentile3 = 99      # Value for the 3rd percentile to track in the reports, the console summary and Graphite
+      percentile4 = 99.9    # Value for the 4th percentile to track in the reports, the console summary and Graphite
+    }
+  }
+  data {
+    writers = [console, file, graphite] # The list of DataWriters to which Gatling write simulation data (currently supported : console, file, graphite, jdbc)
```

Comment:
Ah, I see. My misunderstanding was not with how to establish a TCP connection, but rather with the behavior of iptables. I guess I thought iptables would handle the connection with the foreign host, then create a new connection with the local service if allowable based on the rules. It appears that iptables does not take such a complex approach (and I'm not even sure if it's a feasible approach), and instead literally just drops all packets. So iptables returns nothing. This is in contrast to the same server not running iptables, which would return a RST/SYN if a port was not bound or a SYN/ACK if it was.

Of course, iptables can be configured to return a RST/SYN or some other ICMP packet if desired (using REJECT instead of DROP). But that is neither here nor there.

So I was thinking iptables pulled a Repose. That is, Repose manages a connection to the client (through whatever Servlet container) and a separate connection to the origin service (through Apache HTTPClient). Of course, iptables would have to do that at the transport layer, not the application layer. Again, I'm not sure if that's possible... maybe, since iptables uses the kernel firewall.

I  am really not a network person.